README.md

```diff
@@ -37,17 +36,78 @@ Reverse geocoding is the opposite: returning a list of places near a given point
 - Flexible software architecture
 - Easy to contribute software patches and features to
 - Easy to set up and configure your own instance
-- No external dependencies (such as postgres)
+- No external dependencies (such as PostgreSQL)
```

Comment:
I'm not sure this is really a valid goal (otherwise ElasticSearch would be out too). Perhaps more like "no additional servers / services running", but that isn't quite doing it for me. Maybe the real goal is already handled by the previous line - it just needs to be easy to set up and configure.

Reply:
I reworked this whole section. I think it's much better now.